.github/workflows/compile.yml

```
@@ -16,21 +16,40 @@ permissions:
 jobs:
   build:
     runs-on: ubuntu-latest
+    name: Build TypeScript
 
     steps:
     - name: Checkout repository
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)